man/systemd.network.xml

```diff
@@ -1386,6 +1386,15 @@
             read via <function>sd_network_link_get_sip_servers()</function> function.</para>
           </listitem>
         </varlistentry>
+        <varlistentry>
+          <term><varname>UsePOP3Server=</varname></term>
+          <listitem>
+            <para>When true (the default), the POP3 servers received
+            from the DHCP server will be saved at the state files and can be
+            read via <function>sd_network_link_get_pop3_servers()</function> function.</para>
```

Comment:
Do we need to make this conditional? I'm pretty sure we won't use that for anything internally, so we can simply always export the data, and the external consumer can use it or not as they see fit.